build-system/compile/bundles.config.js

```diff
@@ -754,7 +754,7 @@ exports.extensionBundles = [
   {
     name: 'amp-next-page',
     version: ['0.1', '1.0'],
-    latestVersion: '0.1',
+    latestVersion: '1.0',
```

Comment:
The PR title refers to v2, but the new version number is 1.0. Is that a) intentional and b) desirable?

Reply:
Yes! V2 refers to the second version (the band of the project) but the actual version number is `1.0`